File: 3347-distribute-elements-into-two-arrays-i/3347-distribute-elements-into-two-arrays-i.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var resultArray = function(nums) {
    
    let c=[],d=[]
    c[0]=nums[0]
    d[0]=nums[1]
    for(let i=2;i<nums.length;i++)
    {
        if(c[c.length-1]>d[d.length-1])
            c.push(nums[i])
        else
            d.push(nums[i])
    }
    return[...c,...d]

};

File: 3347-distribute-elements-into-two-arrays-i/3347-distribute-elements-into-two-arrays-i.cs
public class Solution {
    public int[] ResultArray(int[] nums) {
        List<int> arr1 = new List<int>();
        List<int> arr2 = new List<int>();

        arr1.Add(nums[0]);
        if (nums.Length > 1)
        {
            arr2.Add(nums[1]);
        }

        for (int i = 2; i < nums.Length; i++)
        {
            if (arr1[arr1.Count - 1] > (arr2.Count > 0 ? arr2[arr2.Count - 1] : int.MinValue))
            {
                arr1.Add(nums[i]);
            }
            else
            {
                arr2.Add(nums[i]);
            }
        }

        // Combine arr1 and arr2
        arr1.AddRange(arr2);

        return arr1.ToArray();
    }
}